loguru.hpp

```diff
@@ -538,6 +543,15 @@ namespace loguru
 	LOGURU_EXPORT
 	bool add_file(const char* path, FileMode mode, Verbosity verbosity);
 
+#ifdef __linux__
```

Comment:
I believe you originally had this as "not windows". In particular, I tested this on Mac and it worked, so it would be great if it kept working on mac.

I suggest in the header you do something like this:

``` C
#define LOGURU_SYSLOG (defined(__linux__) || defined(__APPLE__))
```

and then do

```
#if LOGURU_SYSLOG
```

everywhere else

Reply:
done